index.php

```diff
@@ -0,0 +1,311 @@
+                        </div>
+                    </li>
+                    <li class="nav-item dropdown mx-1">
+                        <a class="nav-link text-center" href="#" id="dropdownId" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                            <span class="dropdown-link">KIP</span> 
+                            <br>
+                            <i class="fa fa-caret-down text-warning caret-link" aria-hidden="true"></i>
+                        </a>
+                        <div class="dropdown-menu text-center" aria-labelledby="dropdownId">
+                            <a class="dropdown-item" href="index.php?page=regulasi-keterbukaan-informasi-publik">REGULASI KETERBUKAAN INFORMASI PUBLIK                            </a>
+                            <a class="dropdown-item" href="index.php?page=informasi-berkala">INFORMASI BERKALA</a>
+                            <a class="dropdown-item" href="index.php?page=informasi-serta-merta">INFORMASI SERTA MERTA</a>
+                            <a class="dropdown-item" href="index.php?page=informasi-setiap-saat">INFORMASI SETIAP SAAT</a>
+                            <a class="dropdown-item" href="index.php?page=prosedur-permohonan-informasi">PROSEDUR PERMOHONAN INFORMASI</a>
+                            <a class="dropdown-item" href="index.php?page=prosedur-pengajuan-keberatan">PROSEDUR PENGAJUAN KEBERATAN</a>
+                            <a class="dropdown-item" href="index.php?page=prosedur-penyelesaian-sengketa-informasi">PROSEDUR PENYELESAIAN SENGKETA INFORMASI</a>
+                            <a class="dropdown-item" href="index.php?page=laporan-layanan-informasi-publik">LAPORAN LAYANAN INFORMASI PUBLIK</a>
+                        </div>
+                    </li>
+                    <li class="nav-item dropdown mx-1">
+                        <a class="nav-link text-center" href="#" id="dropdownId" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                            <span class="dropdown-link">PENGADAAN</span> 
+                            <br>
+                            <i class="fa fa-caret-down text-warning caret-link" aria-hidden="true"></i>
+                        </a>
+                        <div class="dropdown-menu text-center" aria-labelledby="dropdownId">
+                            <a class="dropdown-item" href="#">E_PROCUREMENT</a>
+                            <a class="dropdown-item" href="index.php?page=procurement-info">INFORMASI PENGADAAN</a>
+                        </div>
+                    </li>
+                    <li class="nav-item mx-1">
+                        <a class="nav-link nav-singgle" href="index.php?page=contact-us">HUBUNGI</a>
+                    </li>
+                </ul>
+            </div>
+        </nav>
+
+        <!-- Include Disini -->
+        <?php
+            include("page/".$_GET["page"].".php");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Detected usage of a non-sanitized input variable: $_GET["page"]](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5431765)